.gitignore

```diff
@@ -1,10 +1,9 @@
 
-src/Setup.java
 config.properties
-out/production/C102/Setup.class
-src/C102_Runner.java
-out/production/C102/C102_Runner.class
-src/C102_RunnerTest.java
-out/production/C102/C102_RunnerTest.class
+out/
```

Comment:
please update the .gitignore with any needed files to ignore.
i.e. command line should only show files that actually need updating